### ISSUE TITLE
AS0094 Info treated as warning > failing pipelines

### ISSUE THREAD

**Issue by fvet** [HodorNV/ALOps#381]
**Describe the bug**
Following #356 we've configured our pipelines to **fail on warni,gs**. 
The solution to resolve the AL0432 warning reported in #356 was to create a rule to demote from **warning** to **info**.

However, since switching to BC 18.1 / runtime 7.0, I now face new failing pipelines on **AS0094**. I decided to demote from **warning** to **info** again (similar to AL0432), but still have my pipelines reporting this as a warning.

**Ruleset.json**

 _{
            "id": "AS0094",
            "action": "Info",
            "justification": "Use the dedicated AL object PermissionSet. Will be managed in a next release (Demoted from warning to Info to avoid pipeline failures)"
        }_

**Compiler output**
##[warning] > shown in orange > warning > pipeline failure

```
*** AL Compiler Exit Code: 0 
*** Set Pipeline variable $(ALOPS_COMPILE_ARTIFACT) = 'C:\DockerInstallation\Agent\_work\189\a\Navitrans_Navitrans 365 Core_18.0.0.13237_sandbox_18.1_w1.app' 
*** Set Pipeline variable $(ALOPS_COMPILE_ARTIFACT_ARRAY) = 'C:\DockerInstallation\Agent\_work\189\a\Navitrans_Navitrans 365 Core_18.0.0.13237_sandbox_18.1_w1.app' 
*** Start processing compile results. 
##[warning]info AS0094: The XML file 'C:\DockerInstallation\Agent\_work\189\s\app\extensionsPermissionSet.xml' should not contain Permissions or Permission Sets. Instead, use the dedicated AL object 'PermissionSet'. 
*** Skip Uploading App as Build-Artifact 
*** Publish XLIF as Artifact skipped 
##[warning] Compile Succeeded with Warnings 

```

**Ruleset.json**

Changing the action to 'None' instead of 'Info' solves the issue, although it's strange info messages are considered as warnings, letting our pipelines fail.

 _{
            "id": "AS0094",
            "action": "None",
            "justification": "Use the dedicated AL object PermissionSet. Will be managed in a next release (Demoted from warning to Info to avoid pipeline failures)"
        }_

Any hints?

**Comment by AdminHodor** [comment]
Dear @fvet ,

We also noticed the output from BC 18 differs a little from the standard on certain errors/warnings. 
A rework of the parser is already on it's way, which we probably release soon as a hotfix on v1.443.

Kind regards,

**Comment by AdminHodor** [comment]
Dear @fvet ,

Could please try again, should be fixed in v1.444

Kind regards,
